scripts/link-subprocess-logs.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+# link subprocess logs for CI to help debug test failures
+
+# This only works for mac-os and linux
+
+if [ -z ${TMPDIR+x} ] ;
+then TMPDIR="/tmp/" ;
+else false ;
+fi
+
+COUNTER=0 ;
+for file in `find $TMPDIR  -name "coda-prover.log"` ;
```

Comment:
Do the names of the containing directories offer any information about the source of the logs?

The names `.1`, `.2`, etc. might not be enough to indicate where the logs came from.

Reply:
The names of the containing directories are randomly generated.